lib/go/templates/templates.go

```diff
@@ -0,0 +1,30 @@
+package templates
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go-sdk"
+)
+
+//go:generate go run github.com/kevinburke/go-bindata/go-bindata -prefix ../../../ -o internal/assets/assets.go -pkg assets -nometadata -nomemcopy ../../../scripts/... ../../../transactions/...
+
+var (
+	placeholderMigrationContractStagingString = "\"MigrationContractStaging\""
+)
+
+func replaceAddresses(code string, migrationContractStagingAddress flow.Address) []byte {
```

Comment:
since this lives inside the generate `templates/templates.go`, id probably rename this to `replaceMigrationContractStagingImports` so its less generic